### PR TITLE
Lab 8

### PR DESCRIPTION
**Pull request by danylykkk**

### CODE REVIEW

File: P8.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import os
+
+class Directory:
```
Comment (polyakovartem) [inline P8.py]: the class name is not informative

File: P8.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import os
+
+class Directory:
+    def __init__ (self, dir:str):
+        self.dir = dir
+
+    def files_list (self):
+        return os.listdir(self.dir)
+
+    def get_duplicates (self):
```
Comment (polyakovartem) [inline P8.py:17]: logic is too complex for one function, split it please

**Review by polyakovartem — CHANGES_REQUESTED** [review]
the solution is suboptimal, please come up with a better one

**Review by polyakovartem — CHANGES_REQUESTED** [review]
could you kindly explain why you did not fix other comments?

**Comment by danylykkk** [comment]
What other conments? Suboptimal solution? What does it mean? Other algorithm?

**Comment by polyakovartem** [comment]
> What other conments? Suboptimal solution? What does it mean? Other algorithm?

well, for instance, the following:
complex logic,
ambiguous class names, etc

please switch to conversation tab and review all of the comments

**Review by polyakovartem — CHANGES_REQUESTED** [review]
please simplify logic and stop ignoring my comments

**Comment by danylykkk** [comment]
I dont know how to simplify it. It is just algorithm I came up with. Please help me, send the samples may be)

**Comment by danylykkk** [comment]
It works. Try to run it.

**Pull request closed without merging**